bbtools/38.99/README.md

```diff
@@ -0,0 +1,39 @@
+## BBTools
+
+This image implements Brian Bushnell's [BBTools](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/).
+
+### Includes
+- [BBDuk](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/): `bbduk.sh` (or `bbduk2.sh`)
+- [BBMap](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbmap-guide/): `bbmap.sh`
+- [BBMerge](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbmerge-guide/): `bbmerge.sh`
+- [Reformat](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/reformat-guide/): `reformat.sh`
+
```

Comment:
Could you please add bullets for samtools, sambaba, and their respective versions to this list?

I think they are two important additions to the docker image and users may not realize that these are included alongside the bbtools suite.